Dockerfile

```diff
@@ -71,16 +71,17 @@ ENV DJANGO_SETTINGS_MODULE cfgov.settings.production
 ENV DJANGO_STATIC_ROOT ${STATIC_PATH}
 ENV ALLOWED_HOSTS '["*"]'
 
+
+# See .dockerignore for details on which files are included
+COPY --chown=apache:apache . .
+
```

Comment:
This change seems unrelated to the purpose of this PR? Doesn't seem like it would be harmful, though.

Reply:
sorry this leaked in from a bad commit from a separate branch, I've reverted it!